doc/uz801.md

```diff
@@ -0,0 +1,68 @@
+# UZ801
+
+The UZ801 is a 4G/LTE USB modem which is built on top of a Qualcomm Snapdragon 410 (MSM8916, with MDM8916 modem.) It does not have a screen, but it does have LEDs which can be used to signal the same status as the green/red bar on the Orbic hotspot. It uses a custom Android-based firmware with limited coreutils. More information about this device can be found [here](https://github.com/AlienWolfX/UZ801-USB_MODEM/wiki/Overview)
+
+It is worth noting that even though the Snapdragon 410 is a quad-core SoC, the CPU has only 2 of the cores enabled on the stock Android-based firmware, probably to avoid overheating as they did not exactly engineer any cooling solution. Regardless, even with 2 disabled cores there is plenty of compute overhead. There are 384MB of RAM on the SoC, and 4GB of eMMC in the form of an SK Hynix NAND flash chip located external to the SoC.
+
+Rayhunter has been tested on UZ801 devices with firmware supporting USB debugging backdoor access. It is not certain whether all of the sticks that use this board will be compatible with the automated installer, or even with any alternative manual installation method. Please consider sharing your device's firmware version and hardware information [here](https://github.com/EFForg/rayhunter/discussions/479) to help improve compatibility.
+
+## Where to purchase
+
+There are several option to purchase this device:
+1. AliExpress:
+- [1](https://www.aliexpress.us/item/3256808999940005.html)
+- [2](https://www.aliexpress.us/item/3256809191207903.html)
+- [3](https://www.aliexpress.us/item/3256809191207903.html)
+2. eBay:
+- [1](https://www.ebay.com/itm/394512588226)
+- [2](https://www.ebay.com/itm/195655408253)
+- [3](https://www.ebay.com/itm/116678550086)
+3. Amazon:
+- [1](https://www.amazon.com/150Mbps-Adapter-Network-Lightweight-Portable/dp/B0DQC64ZFS)
+- [2](https://www.amazon.com/Heayzoki-Network-Adapter-Wireless-Connection/dp/B0CG4W31M4)
+## Supported bands
+
+The UZ801 supports various LTE bands depending on the specific hardware revision and carrier customization. Check your device specifications for the exact band support.
+
+The most frequent bands found on these devices are LTE bands 1/3/5/8/20. In the US, this means that Verizon's band 5 towers are the only towers that this device could communicate with in its normal usage as an LTE modem. Research on whether Qualcomm diagnostic tools can be used to write new band support into the NVRAM is pending.
+
+## Installing
```

Comment:
this section is confusing, it says to only connect over wifi but to me it's pretty clear that ADB requires USB.

if we say we can get ADB to run over the network (which I think is an option on android in principle), that would be really great though, it would sidestep an entire class of driver problems on Windows.

i also think the information about "USB debugging backdoor" and what the installer does internally should probably be a separate section next to "Device-specific notes" rather than being part of installation instructions

Reply:
Good catch, I think that was leftover from the instructions on how to access the Rayhunter web UI, which I realized wouldn't really be necessary as it would be covered by the overview that applies to all devices.

Reply:
One option I found for ADB over wifi is to somehow run `setprop service.adb.tcp.port 5555`.  There's two ways to do that: prior modification of the scripts on the filesystem (which defeats the purpose and/or causes a cyclic dependency) or the discovery of some CVE-level command injection in one of the pathways (which is more likely than one would think).

I think I have broken ground on the latter, since there are eleventy thousand web UI functions that end up invoking shell commands. If one of them is unescaped, we've got a winner.

Reply:
if you end up being able to run shell commands via HTTP directly then you can launch telnet and sidestep ADB entirely.